config/environments/development.rb

```diff
@@ -58,5 +58,5 @@
 
   config.x.logs_faraday_response = true
 
-  Rails.application.routes.default_url_options[:host] = ENV['HOST'] || 'localhost:3002'
+  Rails.application.routes.default_url_options[:host] = ENV['HOST'] || "localhost:#{ENV.fetch('PORT', 3002)}"
```

Comment:
Fixes a personal annoyance - I run with `PORT=3000` in `.env` and have got tired of always having to correct the start citizen journey url.